client-ng1/src/app/js/routes.js

```diff
@@ -605,6 +605,10 @@
         .resolve({account: getAccount})
         .build())
 
+      .state('cfp2017', new State(USER_ROLES, 'cfp2017', 'js/cfp2017/cfp2017.html')
+        .controller('CfpCtrl2017')
+        .build())
+
```

Comment:
I think that we don't need a new page for that. 
We can add a section on the homepage no ? 
see `client-ng1/src/app/js/home/cf.directive.html`